src/include/daos_errno.h

```diff
@@ -43,143 +43,210 @@ extern "C" {
+	ACTION(DER_IO_INVAL,		(DER_ERR_DAOS_BASE + 14),	\
+	       Input / output buffers cannot match object extents)	\
 	/** Event queue is busy */					\
-	ACTION(DER_EQ_BUSY,		(DER_ERR_DAOS_BASE + 15))	\
+	ACTION(DER_EQ_BUSY,		(DER_ERR_DAOS_BASE + 15),	\
+	       Event queue is busy)					\
 	/** Domain of cluster component can't match */			\
-	ACTION(DER_DOMAIN,		(DER_ERR_DAOS_BASE + 16))	\
+	ACTION(DER_DOMAIN,		(DER_ERR_DAOS_BASE + 16),	\
+	       Domain of cluster component cannot match)		\
 	/** Service should shut down */					\
-	ACTION(DER_SHUTDOWN,		(DER_ERR_DAOS_BASE + 17))	\
+	ACTION(DER_SHUTDOWN,		(DER_ERR_DAOS_BASE + 17),	\
+	       Service should shut down)				\
 	/** Operation now in progress */				\
-	ACTION(DER_INPROGRESS,		(DER_ERR_DAOS_BASE + 18))	\
+	ACTION(DER_INPROGRESS,		(DER_ERR_DAOS_BASE + 18),	\
+	       Operation now in progress)				\
 	/** Not applicable. */						\
-	ACTION(DER_NOTAPPLICABLE,	(DER_ERR_DAOS_BASE + 19))	\
+	ACTION(DER_NOTAPPLICABLE,	(DER_ERR_DAOS_BASE + 19),	\
+	       Not applicable)						\
 	/** Not a service replica */					\
-	ACTION(DER_NOTREPLICA,		(DER_ERR_DAOS_BASE + 20))	\
+	ACTION(DER_NOTREPLICA,		(DER_ERR_DAOS_BASE + 20),	\
+	       Not a service replica)					\
 	/** Checksum error */						\
-	ACTION(DER_CSUM,		(DER_ERR_DAOS_BASE + 21))	\
+	ACTION(DER_CSUM,		(DER_ERR_DAOS_BASE + 21),	\
+	       Checksum error)						\
 	/** Unsupported durable format */				\
-	ACTION(DER_DF_INVAL,		(DER_ERR_DAOS_BASE + 22))	\
+	ACTION(DER_DF_INVAL,		(DER_ERR_DAOS_BASE + 22),	\
+	       Unsupported durable format)				\
 	/** Incompatible durable format version */			\
-	ACTION(DER_DF_INCOMPT,		(DER_ERR_DAOS_BASE + 23))	\
+	ACTION(DER_DF_INCOMPT,		(DER_ERR_DAOS_BASE + 23),	\
+	       Incompatible durable format version)			\
 	/** Record size error */					\
-	ACTION(DER_REC_SIZE,		(DER_ERR_DAOS_BASE + 24))	\
+	ACTION(DER_REC_SIZE,		(DER_ERR_DAOS_BASE + 24),	\
+	       Record size error)					\
 	/** Used to indicate a transaction should restart */		\
-	ACTION(DER_TX_RESTART,		(DER_ERR_DAOS_BASE + 25))	\
+	ACTION(DER_TX_RESTART,		(DER_ERR_DAOS_BASE + 25),	\
+	       Used to indicate a transaction should restart)		\
 	/** Data lost or not recoverable */				\
-	ACTION(DER_DATA_LOSS,		(DER_ERR_DAOS_BASE + 26))	\
+	ACTION(DER_DATA_LOSS,		(DER_ERR_DAOS_BASE + 26),	\
+	       Data lost or not recoverable)				\
 	/** Operation canceled (non-crt) */				\
-	ACTION(DER_OP_CANCELED,		(DER_ERR_DAOS_BASE + 27))	\
+	ACTION(DER_OP_CANCELED,		(DER_ERR_DAOS_BASE + 27),	\
+	       Operation canceled)					\
 	/** TX is not committed, not sure whether committable or not */	\
-	ACTION(DER_TX_BUSY,		(DER_ERR_DAOS_BASE + 28))
+	ACTION(DER_TX_BUSY,		(DER_ERR_DAOS_BASE + 28),	\
+	       TX is not committed)
 
 /** Defines the gurt error codes */
 #define D_FOREACH_ERR_RANGE(ACTION)	\
 	ACTION(GURT,	1000)		\
 	ACTION(DAOS,	2000)
 
-#define D_DEFINE_ERRNO(name, value) name = value,
+#define COMMA() ,
```

Comment:
(style)  space prohibited before that ',' (ctx:WxE)
(style)  Macros with complex values should be enclosed in parentheses